### DISCUSSION TITLE
单纯想发点东西

### DISCUSSION THREAD

**Discussion by kaijinti** [share121/inter-knot#628] [其他]
![1722840583855fde3ab1a2662dc580d0d1363c9a4d581abae71d1e413bf56a1d9d99c233518b0 0](https://github.com/user-attachments/assets/31e834ac-a1b5-41c0-bf4c-d70e5609d6cc)
要是自己的水平能再高点就好了

**Comment by hjejdjdj** [comment]
你这个图片怎么发的

**Reply by kaijinti** [reply]
回复的右下角可以插入图片past,drop,or click to add files
